docs/pages/database-access/troubleshooting.mdx

```diff
@@ -39,3 +39,27 @@ details and examples:
 - [Redis Cluster](./guides/redis-cluster.mdx#step-46-set-up-mutual-tls)
 
 After the new certificate is issued, update your database to make it take effect.
+
+### Access to db denied
+
+Attempts to connect to the database fail with an error message
+similar to: **"access to db denied"**.
+
+**Solution:** Configure the database instance and Teleport role-based access control (RBAC) to allow the user 
+access.
+
+Check that the database user and database name (aka schema) exist in the database instance.
+When referring to "database name" or `db_names`, this is not the same as the `name` field in `db_service`;
+this is referring to database names or schemas within a particular database instance.
+When connecting to a PostgreSQL or MongoDB instance, be sure to specify `--db-name`:
+```sh
+$ tsh db connect --db-user=exampleuser --db-name=exampledb
+```
+
+Once your database instance is configured, Teleport RBAC must be configured to allow access.
+Assign the Teleport user a role which **allows** `db_users`, `db_names`, and `db_labels` matching the 
+database user, database name, and Teleport static or dynamic labels for the database. Additionally, check that 
+the user does not have role(s) which **deny** corresponding `db_users`, `db_names`, or `db_labels`.
+For example: 
+
+(!docs/pages/includes/database-access/create-user.mdx!)
```

Comment:
I'm not sure this partial really fits here. We don't need the flag descriptions, and the "Create a local Teleport user..." paragraph doesn't flow very smoothly from the preceding paragraph. What would you think of copying/pasting the partial and using only the relevant parts?

Reply:
I think you make good points. And actually, I think if a user gets directed to this page by an error then their Teleport user already exists - they don't need to create a new one.

I've updated the doc to provide a simple example in-line with a link to RBAC docs at the end